app/controllers/idv/sessions_controller.rb

```diff
@@ -31,6 +36,19 @@ def update
 
     private
 
+    def validation_errors
+      @_validation_errors ||= []
```

Comment:
I have seen it done for "private" variables as a way of saying "hands off for direct access". It's only a convention and doesn't mean anything to the compiler, only to the programmer.